### DISCUSSION TITLE
Custom field issue

### DISCUSSION THREAD

**Discussion by zidaank** [LiveHelperChat/livehelperchat#2103] [General]
Hi I am having an issue with my custom fields. 
 LHCChatOptions.attr.push({
        'name': 'ID Number',
        'value': '<?php echo $user->name?>',
        'type': 'text',
        'size': 6,
        'req': false,
        'show': 'off'

    });
This is my custom field, I would Like the ID number to display here:
![image](https://github.com/LiveHelperChat/livehelperchat/assets/62093822/5775de01-ce76-464d-9445-52d64587af18)
Under the additional data section once a chat is opened but I do not want this field to display on the pop up widget for Online and offline chats. How can I do this?

**Comment by remdex** [comment]
Hi,

First of all please use new syntax for variables passing if you are passing it yourself.
https://doc.livehelperchat.com/docs/custom-fields-and-prefill

You can't have different fields for popup and mormal chatrs. For offline and offline form you can just use.
![image](https://github.com/LiveHelperChat/livehelperchat/assets/1146085/72a53894-1174-4d37-a17e-008201abae83)
